modules/nf-core/samshee/environment.yml

```diff
@@ -0,0 +1,8 @@
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - python=3.12.7
+  - pip
+  - pip: # FIXME https://github.com/nf-core/modules/issues/5814
+      - samshee==0.2.1
```

Comment:
Ask in slack what we can do with this as we will need a core team member to approve this PR